project-base/storefront/urql/cacheExchange.ts

```diff
@@ -146,23 +154,21 @@ export const cache = cacheExchange({
                         : undefined;
                 manuallyUpdateCartFragment(cache, newCart);
             },
-            addProductToComparison(result, _args, cache) {
-                invalidateFields(cache, ['comparison']);
-            },
             removeProductFromComparison(result, _args, cache) {
```

Comment:
In here, but also in all the functions bellow, you should use `args` instead of `_args`. The underscore is only if you're ignoring the parameter.

Reply:
Oh, I didn't know that. Thanks. Renamed in 42265a06b1